module3/projects/consultancy/ideation.md

```diff
@@ -0,0 +1,170 @@
+```
+
+### APIs
+
+Your application **must make good use of at least one external dataset or API**. Some examples include:
+
+#### Government Data
+
+* [Data.gov](https://www.data.gov/)
+* [ProPublica](http://www.propublica.org/tools/)
+* [NASA](http://data.nasa.gov/api-info/)
+* [US Census](http://www.census.gov/data/developers/data-sets.html)
+* [Socrata Listings](https://opendata.socrata.com/dataset/Socrata-Customer-Spotlights/6wk3-4ija)
+* [Bureau of Labor & Statistics](http://www.bls.gov/developers/api_ruby.htm)
+* [United Nations](https://www.undata-api.org/) (3rd party API)
+* [Google's Directory of Public Data](http://www.google.com/publicdata/directory)
+* [OpenColorado](http://data.opencolorado.org/)
+* [Denver Regional Council of Governments](https://drcog.org/services-and-resources/data-maps-and-modeling)
+
+#### Corporate Data
+
+* [Facebook](https://developers.facebook.com)
+* [Instagram](https://instagram.com/developer)
+* [Github](https://developer.github.com/v3)
+* [FitBit](https://dev.fitbit.com)
+* [Spotify](https://developer.spotify.com/web-api)
+* [Strava](https://www.strava.com/developers)
+* [Google Maps](https://developers.google.com/maps)
+
+The following links are catalogs of APIs on rapidapi.com. You will not implement RapidAPI, it's just a great place to find other APIs.
+
+free/discounted apis for students:
+- [https://rapidapi.com/collection/rapidapi-school](https://rapidapi.com/collection/rapidapi-school)
+
+free public APIs:
+- [https://rapidapi.com/collection/list-of-free-apis](https://rapidapi.com/collection/list-of-free-apis)
+
+covid-19 apis:
+- [https://rapidapi.com/collection/coronavirus-covid-19](https://rapidapi.com/collection/coronavirus-covid-19)
+
+job postings and job search data:
+- [https://rapidapi.com/collection/job-search-apis](https://rapidapi.com/collection/job-search-apis)
+
+fun apis like english-to-yoda:
+- [https://rapidapi.com/collection/cool-apis](https://rapidapi.com/collection/cool-apis)
+
+google-specific apis:
+- [https://rapidapi.com/collection/google-api](https://rapidapi.com/collection/google-api)
+
+online databases of collections of some sort of data (ie, movies, music, etc):
+- [https://rapidapi.com/collection/database-apis](https://rapidapi.com/collection/database-apis)
+
+alternatives to Google Maps:
+- [https://rapidapi.com/collection/alternatives-to-google-maps-api](https://rapidapi.com/collection/alternatives-to-google-maps-api)
+
+Other API Sources:
+
+- [https://apilist.fun/](https://apilist.fun/)
+- [https://github.com/public-apis/public-apis](https://github.com/public-apis/public-apis)
+- [https://api.data.gov/list-of-apis/](https://api.data.gov/list-of-apis/)
+
+However, the list is not limited to these. You can choose to integrate with a service of your choosing, as long as it is approved ahead of time.
+
+---
+
+## Some Project Ideas to get the Creative Process Started
+
+### Previous Student Pitches
```

Comment:
Love this too! I think students get overwhelmed trying to think of ideas. It's great giving them examples of appropriate scope for this project.